### PR TITLE
[SITE-303] Site Details Page & Cards

### PR DESCRIPTION
**Pull request by bswatson**
This PR covers multiple tickets:

* [SITE-303: BE: Site Details Card Connections](https://moderntribe.atlassian.net/browse/SITE-303)
* [SITE-309: BE: Your Content card](https://moderntribe.atlassian.net/browse/SITE-309)
* [SITE-300: BE: Store FTC Wizard Connections](https://moderntribe.atlassian.net/browse/SITE-300)
* [SITE-366: Manage Your Products Card](https://moderntribe.atlassian.net/browse/SITE-366)

It is based on #223 and should either be merged into that branch or if that branch is merged first, then the PR should be updated and merged directly to main once completed.

The changes in this PR are available to test on https://0020f93ddd.nxcli.io/

### CODE REVIEW

**Review by dotjon — APPROVED** [review]

**Comment by bswatson** [comment]
@aykanburcak wanted to call your attention to `plugins/wme-sitebuilder/wme-sitebuilder/Cards/Shipping.php` which duplicates and conflicts with your work on #236.

**Comment by aykanburcak** [comment]
> @aykanburcak wanted to call your attention to `plugins/wme-sitebuilder/wme-sitebuilder/Cards/Shipping.php` which duplicates and conflicts with your work on #236.

Looks like the only addition on my PR is the footer links of shipping card.
I can rebase my branch after you merge this.
Does it sound ok to you @bswatson ?

**Comment by bswatson** [comment]
Sounds great! I'll get this merged and then we can wrap that up.

File: plugins/wme-sitebuilder/wme-sitebuilder/Cards/Goals.php [hunk 1]
```diff
@@ -59,9 +59,9 @@ public function maybe_add_store_builder_task( $rows ) {
 		$rows[] = [
 			'id'      => 'goals-woocommerce',
 			'type'    => 'task',
-			'taskCta' => __( 'Get Started', 'wme-sitebuilder' ),
-			'title'   => __( 'Set up your store', 'wme-sitebuilder' ),
-			'intro'   => __( 'Set up your store.', 'wme-sitebuilder' ),
+//			'taskCta' => __( 'Get Started', 'wme-sitebuilder' ),
```
Comment (aykanburcak) [inline plugins/wme-sitebuilder/wme-sitebuilder/Cards/Goals.php:62]: This can be removed if not necessary.

Comment (bswatson) [reply plugins/wme-sitebuilder/wme-sitebuilder/Cards/Goals.php]: Good catch! I removed it.

**Pull request merged**